exampleSite/data/navigation.toml

```diff
@@ -0,0 +1,3 @@
+[[links]]
+title = "About"
+url = "/about/"
```

Comment:
Should this file exist in the `/en` folder instead?

With this structure you cannot have different translations of "About" for example depending on which language you're viewing the site in.